lib/enumerize/attribute.rb

```diff
@@ -17,6 +17,13 @@ def initialize(klass, name, options={})
         @default_value = find_default_value(options[:default])
         raise ArgumentError, 'invalid default value' unless @default_value
       end
+
+      # Define methods to get each value
+      @values.each do |v|
+        metaclass = class << self; self; end
+        metaclass.send(:define_method, v.to_s) { v.value }
+        metaclass.send(:define_method, v.to_s + '_text' ) { v.text }
+      end
```

Comment:
There was an issue with predicate methods on `Enumerize::Value` object #99. The `Enumerize::Value` objects store references to the `Enumerize::Attribute` objects so I guess defining methods on a `Enumerize::Attribute` singleton class will make `Enumerize::Value` unmarshallable causing the same issue as #99.

I'm thinking about defining named `Enumerize::Attribute` subclasses (this will solve this `Marshal` issue) but I'm not sure about it right now.